samples/YesSql.Samples.FullText/Program.cs

```diff
@@ -13,7 +14,7 @@ private static void Main(string[] args)
         {
             var configuration = new Configuration
             {
-                ConnectionFactory = new DbConnectionFactory<SqliteConnection>(@"Data Source=:memory:", true),
+                ConnectionFactory = new DbConnectionFactory<MySqlConnection>(@"server=127.0.0.1,3306;uid=root;pwd=920624ppN_;database=dbSample2;", true),
```

Comment:
Probably not wise to change this, the goal of this app is not really to show how to do full text indexing, and Sqlite is more adapted as it requires no configuration.